src/_components/footer.ejs

```diff
@@ -1,22 +1,21 @@
 <div class="c-download-banner">
     <div class="container">
-        <h2>Скачайте приложение и начните свое увлекательное путешествие по Беларуси </h2>
+        <h2><%= lang[locale].downloadText %></h2>
         <div class="c-download-banner__btns">
             <a href="<%= site.appGooglePlay %>" target="_blank" rel="noopener">
-                <img alt="Get it on Google Play" src="/images/app-google-white.png" width="167" height="48">
+                <img alt="Get it on Google Play" src="/images/app-google-play.svg" width="167" height="48">
```

Comment:
It might be a11y-friendly to localized alt text, so that it's not always `Get it on Google Play`.

Reply:
Updated to use translations